components/Header.js

```diff
@@ -1,91 +1,154 @@
-import React, { Component } from 'react'
```

Comment:
Why so many changes are made to this file? Just the link has to be added for faq.
Let me know if you have any questions.